node/middlewares/scheduler/deleteScheduler.ts

```diff
@@ -0,0 +1,26 @@
+import { CRON } from '../../utils/constants'
+import { formatError } from '../../utils/formatError'
+
+export async function deleteScheduler(ctx: Context, next: () => Promise<any>) {
+    const {
+      clients: { scheduler },
+      vtex: { logger },
+    } = ctx
+  
+    try {
+      await scheduler.deleteScheduler(CRON.id)
+  
+      ctx.status = 200
+      ctx.body = 'Cron deleted'
+  
+      await next()
+    } catch (e) {
+      logger.error({
+        middleware: 'DELETE SELLER CRON',
```

Comment:
Can you please change the string of middleware from 'DELETE SELLER CRON', into something related with this project, like 'DELETE RETURN APP CRON'.